src/CozyUtils.js

```diff
@@ -153,30 +153,61 @@ class CozyUtils {
    * @param  {array} files   files received from the toutatice API call
    * @returns {object}
```

Comment:
Now, you don't return the object, you return a Promise that resolve to the object. 

So something like: 
```
@returns {Promise<object>}
``` 

is better. 

You can add `//@ts-check`  at the top of your JS files, it'll help you a lot ;).